meetings/2022-02-28.md

```diff
@@ -0,0 +1,131 @@
+* EP: What about the Client Identifier, it being defined by Solid-OIDC
+* AC: In that case it would make a lot of sense for it to be required to use https
+* EP: The WebID spec doesn't specify, so in this case is it bad to add our own requirements on top, narrowing it of some sorts, like requiring the trusted OP to be referred?
+
+### Primer: multiple issues
+
+[Issue#83](https://github.com/solid/solid-oidc/issues/83)
+
+* EP: Contributions for [Dimitri Teleguin](https://github.com/dteleguin) (i believe keycloak contributor). If anyone can help him contribute that would be great.
+* AC: Indeed, the primer has been neglected. I'll open some small PRs to try and address some issues.
+* EP: Very well, i'll focus on #65. KD, do you have feedback on the primer and spec ?
+* KD: Not necessarily, I'm learning a lot still.
+
+### Organize sections by conformance classes
+
+[Issue#65](https://github.com/solid/solid-oidc/issues/65)
+
+* EP: First step with PR#79 for the WebID profile document. Similar example is the ID token verification, which happens on both the client side and the AS too. How do we want to eventually add the test suite annotations on the conformance classes ? In the case of multiple parties who need to comply with a certain part, eg ID token verification. Could it be identified as an atomic requirement, applied to multiple conformance classes ?
+* AC: This kind of spec organization will make things easier for implementers and people testing implementations. Because there are a number of conceptual entites involved, each having its own responsibilities. Unless you understand each of these components, it is confusing who is responsible for what. The new organization intends at reducing this confusion. KD, since you're looking at implementing an OP your feedback will be very valuable.
+* KD: One of the recent changes made the sequence diagram noticeably more complex. We still have a lot of areas to investigate to figure out what responsibilites our software will have.
+* AC: It's really great that you're asking the questions to the panel.
+* EP: ID token verification is the next step I want to address, without specifying who is doing the verification. Maybe describing a mixin class "OIDC token verifier", and refer to it for different conformance classes
+* NS: it would be useful to have these mixin classes, but it might be interesting to consider whether that verification should happen differently, depending on the party. e.g., certain claims may need to be validated differently. May also want to consider having a primary conformance class for how an ID token should be validated, then other conformance classes could refer to that conformance class.
```

Comment:
```suggestion
* NS: it would be useful to have these mixin classes, but it might be interesting to consider whether that verification should happen differently, depending on the party. For instance, certain claims may need to be validated differently. May also want to consider having a primary conformance class for how an ID token should be validated, then other conformance classes could refer to that conformance class.
```